src/windows/CameraProxy.js

```diff
@@ -370,10 +395,11 @@ function takePictureFromCameraWP(successCallback, errorCallback, args) {
             if (sensor !== null) {
                 sensor.addEventListener("orientationchanged", onOrientationChange);
             }
-            capturePreview.addEventListener('click', captureAction);
+            //capturePreview.addEventListener('click', captureAction);
```

Comment:
Am I suppose to edit my file with these changes? Never had a "nit" before
On 9 Jul 2015 7:31 pm, "Murat Sutunc" <notifications@github.com> wrote:

> In src/windows/CameraProxy.js
> <https://github.com/apache/cordova-plugin-camera/pull/106#discussion_r34287483>
> :
>
> > @@ -370,10 +395,11 @@ function takePictureFromCameraWP(successCallback, errorCallback, args) {
> >              if (sensor !== null) {
> >                  sensor.addEventListener("orientationchanged", onOrientationChange);
> >              }
> > -            capturePreview.addEventListener('click', captureAction);
> > +            //capturePreview.addEventListener('click', captureAction);
>
> nit: remove comment if not necessary
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/cordova-plugin-camera/pull/106/files#r34287483>
> .
>